src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -155,6 +155,7 @@ public void addGroup(Group group) {
     public void deleteGroup(Group group) {
         requireNonNull(group);
         addressBook.deleteGroup(group);
+        updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
```

Comment:
Maybe this can be omitted, or moved to the relevant GroupDeleteCommand class? Any updates to lists that are observable by the UI will be triggered in the execute() functions of the Command classes.

Reply:
I added this initially because the groups in the person panel did not refresh/reflect the changes of the deleted group